pcsx2/CMakeLists.txt

```diff
@@ -953,6 +958,10 @@ if(PORTAUDIO_FOUND)
     set(pcsx2FinalLibs ${pcsx2FinalLibs} ${PORTAUDIO_LIBRARIES})
 endif()
 
+if(PULSEAUDIO_FOUND)
+    set(pcsx2FinalLis ${pcsx2FinalLibs} ${PULSEAUDIO_LIBRARIES})
```

Comment:
Probably wanted to use pcsx2FinalLibs there?

Reply:
Thanks, good catch.